pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -70,9 +70,9 @@ func (p *cadvisorStatsProvider) ListPodStats() ([]statsapi.PodStats, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to get rootFs info: %v", err)
 	}
-	imageFsInfo, err := p.cadvisor.ImagesFsInfo()
+	imageFsStats, err := p.ImageFsStats()
```

Comment:
Can you please explain why we need this change?

Reply:
Hm, actually we could keep the original code, and pass `nil` in stats_provider. Good catch

Reply:
Done.